test/decl/typealias/generic.swift

```diff
@@ -1,9 +1,31 @@
-// RUN: %target-parse-verify-swift
+// RUN: %target-parse-verify-swift -enable-experimental-nested-generic-types
 
-struct MyType<TyA, TyB> {
+struct MyType<TyA, TyB> { // expected-note {{declared here}}
   var a : TyA, b : TyB
 }
 
+//
+// Type aliases that reference unbound generic types -- not really generic,
+// but they behave as such, in the sense that you can apply generic
+// arguments to them.
+//
+
+// FIXME: This should work?
+typealias OurType = MyType  // expected-error {{reference to generic type 'MyType' requires arguments in <...>}}
```

Comment:
Unfortunately the qualified lookup case does work, and at least xctest relies on it.

I'd be in favor of making this behavior consistent between the qualified and unqualified cases. It seems like there would be some fallout though.